src/daemon/ta/proxy.rs

```diff
@@ -435,187 +484,186 @@ impl eventsourcing::Aggregate for TrustAnchorProxy {
                 command
             );
         }
+
         match command.into_details() {
-            // Publication Support
-            TrustAnchorProxyCommandDetails::AddRepository(repository) => {
-                if self.repository.is_none() {
-                    Ok(vec![TrustAnchorProxyEvent::new(
-                        &self.handle,
-                        self.version,
-                        TrustAnchorProxyEventDetails::RepositoryAdded(repository),
-                    )])
-                } else {
-                    Err(Error::TaProxyAlreadyHasRepository)
-                }
+            // Initialisation
+            TrustAnchorProxyCommandDetails::Initialise => {
+                // This can't happen really.. we would never send this command
+                // to an existing TrustAnchorProxy.
+                //
+                // This could be solved more elegantly, and more verbosely, if
+                // we create a separate TrustAnchorProxyStorableCommand that
+                // implements 'WithStorableDetails' - like we have in other cases -
+                // because then our initialisation command could map to that type
+                // instead of having this additional variant for storing.
+                Err(Error::custom("Trust Anchor Proxy already created"))
```

Comment:
Same comment as elsewhere as to whether this should say "created" or "initialised".